invenio_rdm_records/fixtures/tasks.py

```diff
@@ -43,9 +46,21 @@ def get_authenticated_identity(user_id):
 
 @shared_task
 def create_vocabulary_record(service_str, data):
-    """Create a vocabulary record."""
+    """Create or update a vocabulary record."""
     service = current_service_registry.get(service_str)
-    service.create(system_identity, data)
+    if "type" in data:
```

Comment:
this might fail if the custom vocabulary has a `type` field, but at this point we do not have any other more sensible way of checking this.